kie-server-parent/kie-server-services/kie-server-services-jbpm-ui/src/main/java/org/kie/server/services/jbpm/ui/form/render/AbstractFormRenderer.java

```diff
@@ -379,14 +405,35 @@ protected void processFormLayout(FormInstance topLevelForm,
                                 value = outputs.get(field.getBinding());
                             }
                             
-                            item.setValue(value);
+                            if(!fieldsExclusionTransformation.contains(fieldType)) {
+                                item.setValue(value.toString());
+                            } else {
+                                DocumentCollection<Document> docColletion = (DocumentCollection<Document>) value;
```

Comment:
you setup this wrong. The type of the field should be org.jbpm.document.DocumentCollection
and it should be binded to a document collection. Otherwise it is not possible to make it work.
I am adding a instanceof check anyway.